apps/projects/app/components/Form/Input/DropDownOptionsInput.js

```diff
@@ -24,48 +25,149 @@ class DropDownOptionsInput extends React.Component {
       )
     }
   }
+  
+  state = {
+    showAddOption: false,
+    addOptionText: '',
+    found: [],
+  }
+
+  addOption = () => {
+    this.setState({ showAddOption: true })
+  }
+
+  clearState = () =>    this.setState({ showAddOption: false, addOptionText: '', found: [] })
+
+  addToCurated = issue => () => {
+    this.props.onChange({
+      target: { name, value: this.props.values.push(issue) },
+    })
+    this.clearState()
+  }
+
+  searchOptions = ({ target: { name, value } }) => {
+    const found = this.props.allOptions.filter(
+      issue => {
+        if (!issue.title.includes(value) ||''.toString(issue.number).includes(value)) return false
+        return (this.props.values.findIndex(i => i.id === issue.id) === -1)
```

Comment:
if (!issue.title.includes(value) // text search in issue title
||
''.toString(issue.number).includes(value)) // converting issue number to string so it can be text-searched

and this:

this.props.values.findIndex(i => i.id === issue.id) === -1

this makes sure that issues matching text search have not already been added as options for curation... this.props.values is an array containing issues displayed as options.